src/main/scala/com/cognite/sdk/scala/v1/Client.scala

```diff
@@ -6,6 +6,7 @@ package com.cognite.sdk.scala.v1
 import BuildInfo.BuildInfo
 import cats.{Id, Monad}
 import cats.implicits._
+import cats.catsInstancesForId
```

Comment:
Scala 3 needs it, I guess it's because it doesn't import implicits by default in the same way. Good example of the improved error messages:
```
[error] -- Error: /home/wjoel/work/cognite-sdk-scala/src/main/scala/com/cognite/sdk/scala/v1/Client.scala:298:74 
[error] 298 |    extends GenericClient[Id](applicationName, projectName, baseUrl, auth)
[error]     |                                                                          ^
[error]     |Could not find an instance of Monad for cats.Id.
[error]     |I found:
[error]     |
[error]     |    cats.Invariant.catsStdCommutativeMonadForTuple11[A0, A1, A2, A3, A4, A5, A6, A7
[error]     |      , 
[error]     |    A8, A9]
[error]     |
[error]     |But method catsStdCommutativeMonadForTuple11 in trait NTupleMonadInstances1 does not match type cats.Monad[cats.Id].
[error]     |
[error]     |The following import might fix the problem:
[error]     |
[error]     |  import cats.catsInstancesForId
```